web/src/engine/websites/ShojoScans.ts

```diff
@@ -0,0 +1,28 @@
+import { Tags } from '../Tags';
+import icon from './ShojoScans.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as MangaStream from './decorators/WordPressMangaStream';
+import * as Common from './decorators/Common';
+
+function ChapterExtractor(anchor: HTMLAnchorElement) {
+    return {
+        id: anchor.pathname,
+        title: anchor.querySelector('span.chapternum').textContent.trim().replaceAll('\n', '').replaceAll('\t', ''),
+    };
+}
+
+@MangaStream.MangaCSS(/^{origin}\/comics\/[^/]+\/$/, 'h1.entry-title')
```

Comment:
In that context, its kinda useless to use @Mangastream.MangaCSS.
@Common.MangaCSS is doing the same.

Reply:
pushed and tester localy